### PR TITLE
Get by id 19

### LINKED ISSUE #19: getById function
people.getById is needed for auth.js

Requested here:
https://github.com/emfoundation/ce100-app/issues/390

possible interface:
```js
request.pg.people.getById(id, function (err, person) {
```

This function should only ever return 1 or 0 people, I am thinking it could always return an array as it would be very easy to tell if there is a person using `person.length === 0`.

Thoughts @SimonLab ?

### PR DESCRIPTION
**Pull request by jay-meister**
Make getByEmail more reusable

See readme changes or issue #19 for new interface

closes #19

### CODE REVIEW

**Review by SimonLab — COMMENTED** [review]
Looks good @JMurphyWeb, can you just update the version in the package.json, thanks

File: lib/queries/peopleGetBy.js [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 var escape = require('pg-escape');
 
-module.exports = function (email) {
+module.exports = function (constraint, value) {
```
Comment (SimonLab) [inline lib/queries/peopleGetBy.js:5]: We might need to add more validation on the constraint parameter later on to be sure that  the values match one of the column

Comment (jay-meister) [reply lib/queries/peopleGetBy.js:5]: yup.

**Review by SimonLab — APPROVED** [review]
:+1:

**Pull request merged**